### PR TITLE
Clarify getitem docstring. Log when falling back.

### PR DESCRIPTION
**Pull request by RollingStar**
I figure, let someone change the docstring again if the function gets a special case besides artist / album artist.

### CODE REVIEW

File: beets/library.py [hunk 1]
```diff
@@ -410,16 +410,19 @@ def _get(self, key):
             raise KeyError(key)
 
     def __getitem__(self, key):
-        """Get the value for a key. Certain unset values are remapped.
+        """Get the value for a key. `artist` and `albumartist`
+		are fallback values for each other when unmapped.
```
Comment (RollingStar) [inline beets/library.py]: Better wording: "when not set"?

Comment (sampsyo) [reply beets/library.py]: Yeah, good point—"when not set" is better.

**Comment by sampsyo** [comment]
Thanks! The docstring improvement is definitely better. About the logging, however—it seems like this could be pretty verbose! Just accessing a field like `bar = foo.artist` seems like it should probably not have side effects, but this will log every time you look up the field. Using `beet ls -f '$artist'` might print this out thousands of times, for example.

Maybe there's some more centralized place where we can put this logging that won't be quite so intrusive?

**Comment by jtpavlock** [comment]
@RollingStar do you still have interest in continuing work on this?

**Comment by sampsyo** [comment]
Yo; I rolled this back to just the docstring change so we can take that! Always happy to explore different logging in the future.

**Comment by RollingStar** [comment]
Thanks. I'm still a bit of a mess with git.

**Pull request merged**